lib/polar_plotting/polar_plot.dart

```diff
@@ -0,0 +1,201 @@
+import 'package:csv/csv.dart';
+import 'dart:io';
+import 'dart:convert';
+import 'dart:async';
+import 'dart:math';
+import 'dart:collection';
+import 'package:quiver/iterables.dart';
+// Polar plot class generated through csv polar plot files
```

Comment:
What's the format for these csv polar plot files? Would be nice to see documentation for what each column represents, and what multiple rows might mean.